### PR TITLE
Update to jsdom v12

### PR DESCRIPTION
**Pull request by sarvaje**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [x] Added/Updated related tests.

### CODE REVIEW

File: packages/connector-jsdom/package.json [hunk 1]
```diff
@@ -9,8 +9,10 @@
   "dependencies": {
     "@hint/utils-connector-tools": "^1.0.3",
     "canvas-prebuilt": "^1.6.5-prerelease.1",
-    "jsdom": "^11.12.0",
-    "node-pre-gyp": "^0.11.0"
+    "jsdom": "git://github.com/sarvaje/jsdom.git#v12-fixes",
```
Comment (alrra) [inline packages/connector-jsdom/package.json]: Is there an PR for this? If so can you link it here. Thanks!

Comment (sarvaje) [reply packages/connector-jsdom/package.json]: The problem is that I have 2 PR for this :(

Comment (alrra) [reply packages/connector-jsdom/package.json]: Can you link to them here. Thanks!

**Comment by sarvaje** [comment]
This pull request depends on https://github.com/jsdom/jsdom/pull/2339 and https://github.com/jsdom/jsdom/pull/2349. That's why right now the link to jsdom is to my repository.

**Comment by alrra** [comment]
Closing in favor of #1357.

**Pull request closed without merging**